### PR TITLE
Use Head and Torso wording instead of Head and Body

### PR DESCRIPTION
**Pull request by bux**
**When merged this pull request will:**
- title

Could you please check if I translated your langugage correctly?
@IllidanS4 
@TBakerBR 
@D3L7A01 

Does your language reflect the changes? `Body` -> `Torso`
@classicarma 
@dfr1238

### CODE REVIEW

**Comment by D3L7A01** [comment]
technically TORSO for body is correct

**Comment by bux** [comment]
@D3L7A01 
image

File: addons/medical_statemachine/stringtable.xml [hunk 1]
```diff
@@ -22,13 +22,13 @@
             <Italian>Lesioni mortali del giocatore</Italian>
         </Key>
         <Key ID="STR_ACE_Medical_Statemachine_FatalInjuriesPlayer_Description">
-            <English>Controls when players can receive fatal injuries. A fatal injury is caused by significant damage to the head or body.</English>
-            <Portuguese>Controla quando os jogadores podem receber lesões fatais. Uma lesão fatal é causada por um dano significante na cabeça ou corpo.</Portuguese>
+            <English>Controls when players can receive fatal injuries. A fatal injury is caused by significant damage to the head or torso.</English>
+            <Portuguese>Controla quando os jogadores podem receber lesões fatais. Uma lesão fatal é causada por um dano significante na cabeça ou tronco.</Portuguese>
             <Japanese>プレイヤーが致命傷を受けた時の挙動を管理できます。頭部や胴体に受ける大きなダメージは致命傷になります。</Japanese>
```
Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]: We have to replace "胴体" with "胸部". Just replace with this.
"プレイヤーが致命傷を受けた時の挙動を管理できます。頭部や胸部に受ける大きなダメージは致命傷になります。"

File: addons/medical_statemachine/stringtable.xml [hunk 2]
```diff
@@ -40,13 +40,13 @@
...
+            <English>Controls when AI can receive fatal injuries. A fatal injury is caused by significant damage to the head or troso.\nWhen set to "Always", this effectively produces "AI Instant Death" behaviour as AI will immediately die from any fatal injury.\nNOTE: Any mode other than "Always" requires AI Unconsciousness to be enabled.</English>
+            <Portuguese>Controla quando a IA pode receber lesões fatais. Uma lesão fatal é causada por um dano significante na cabeça ou tronco.\nQuando definido para "Sempre", isso efetivamente causa a "Morte Instantânea da IA", pois a IA irá imediatamente morrer para qualquer lesão fatal.\nNOTA: Qualquer opção além de "Sempre" requer que Inconsciência de IA esteja ativada.</Portuguese>
             <Japanese>AI が致命傷を受けた時の挙動を管理できます。頭部や胴体に受ける大きなダメージは致命傷になります。\n"常に"に設定されていると、いかなる致命傷でも "AI の即死" 効果が生まれます。\n注: "常に"以外のモードでは AI の無意識化を有効化させる必要があります。</Japanese>
```
Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]: Same. 
"AI が致命傷を受けた時の挙動を管理できます。頭部や胸部に受ける大きなダメージは致命傷になります。\n"常に"に設定されていると、いかなる致命傷でも "AI の即死" 効果が生まれます。\n注: "常に"以外のモードでは AI の無意識化を有効化させる必要があります。"

**Comment by D3L7A01** [comment]
> @D3L7A01
> image

in Italy we use to say so, however, torso can remain torso and body corpo. we have a particular dialectic in italy  XD

File: addons/medical_statemachine/stringtable.xml [hunk 3]
```diff
@@ -40,13 +40,13 @@
...
+            <Czech>Nastavuje zda AI může utrpět smrtelné zranění. Smrtelné zranění je utrpěto významným poškozením hlavy nebo trupu.\nPokud je tato možnost nastavena na "Vždy", efektivně nastává stav "Povolit okamžitou smrt AI", protože AI okamžitě zemře při jakémkoliv smrtelném zranění.\nPOZNÁMKA: Jakýkoliv jiný stav než "Vždy" potřebuje zapnutou možnost "Bezvědomí AI".</Czech>
+            <Italian>Controlla quando l'IA può ricevere ferite mortali. Una lesione mortale è causata da un danno significativo alla testa o al busto. Se impostato su "Sempre", questo produce efficacemente il comportamento "Morte istantanea AI", poiché l'IA morirà immediatamente a causa di qualsiasi lesione mortale.\NOTA: Qualsiasi modalità diversa da "Sempre" richiede l'attivazione dell'Incoscienza AI.</Italian>
```
Comment (D3L7A01) [inline addons/medical_statemachine/stringtable.xml]: <Italian>Controlla quando l'IA può ricevere ferite mortali. Una lesione mortale è causata da un danno significativo alla testa o al torso. Se impostato su "Sempre", questo produce efficacemente il comportamento "Morte istantanea AI", poiché l'IA morirà immediatamente a causa di qualsiasi lesione mortale.\NOTA: Qualsiasi modalità diversa da "Sempre" richiede l'attivazione dell'Incoscienza AI.</Italian>


line 49 changed

File: addons/medical_statemachine/stringtable.xml (hunk 1 above)
Comment (bux) [reply addons/medical_statemachine/stringtable.xml]: Could you please use the suggestion feature?
image

Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
            <Japanese>プレイヤーが致命傷を受けた時の挙動を管理できます。頭部や胸部に受ける大きなダメージは致命傷になります。</Japanese>
```

File: addons/medical_statemachine/stringtable.xml (hunk 2 above)
Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
            <Japanese>AI が致命傷を受けた時の挙動を管理できます。頭部や胸部に受ける大きなダメージは致命傷になります。\n"常に"に設定されていると、いかなる致命傷でも "AI の即死" 効果が生まれます。\n注: "常に"以外のモードでは AI の無意識化を有効化させる必要があります。</Japanese>
```

File: addons/medical_statemachine/stringtable.xml (hunk 1 above)
Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
             <Japanese>プレイヤーが致命傷を受けた時の挙動を管理できます。頭部や胴体に受ける大きなダメージは致命傷になります。</Japanese>
```

File: addons/medical_statemachine/stringtable.xml (hunk 2 above)
Comment (classicarma) [inline addons/medical_statemachine/stringtable.xml]:
```suggestion
             <Japanese>AI が致命傷を受けた時の挙動を管理できます。頭部や胸部に受ける大きなダメージは致命傷になります。\n"常に"に設定されていると、いかなる致命傷でも "AI の即死" 効果が生まれます。\n注: "常に"以外のモードでは AI の無意識化を有効化させる必要があります。</Japanese>
```

File: addons/medical_statemachine/stringtable.xml (hunk 1 above)
Comment (classicarma) [reply addons/medical_statemachine/stringtable.xml]: Sure and I didn't know that button😄

**Comment by TBakerBR** [comment]
Yup. All green in the portuguese translation.

File: addons/medical_statemachine/stringtable.xml (hunk 3 above)
Comment (bux) [reply addons/medical_statemachine/stringtable.xml]: Could you please use the suggestion feature?
image

**Pull request merged**